website/content/docs/release-notes/v0_15_0.mdx

```diff
@@ -0,0 +1,241 @@
+    <br /><br />
+    As a best practice, administrators should rotate credentials in a phased manner, ensuring that all previously established sessions are completed before revoking the stale credentials.
+    Otherwise, you may end up with recordings that aren't stored in the remote storage bucket, and are unable to be played back.
+    </td>
+  </tr>
+  <tr>
+    <td style={{verticalAlign: 'middle'}}>
+    0.13.0+
+    </td>
+    <td style={{verticalAlign: 'middle'}}>
+    Unsupported recovery workflow during worker failure
+    </td>
+    <td style={{verticalAlign: 'middle'}}>
+    If a worker fails during a recording, there is no way to recover the recording. This could happen due to a network connectivity issue or because a worker is scaled down, for example.
+    <br /><br />
+    Learn more:&nbsp;
+    <a href="/boundary/docs/troubleshoot/troubleshoot-recorded-sessions#unsupported-recovery-workflow">Unsupported recovery workflow</a>
+    </td>
+  </tr>
+
+<tr>
+    <td style={{verticalAlign: 'middle'}}>
+    0.15.0
+    </td>
+    <td style={{verticalAlign: 'middle'}}>
+    Permission grant string change
+    </td>
+    <td style={{verticalAlign: 'middle'}}>
+    The ability to add new grants via the <code>id</code> parameter has been removed. Grants now accept more than one ID per grant string or JSON entry using the <code>ids</code> parameter. This change was noted in the <a href="https://github.com/hashicorp/boundary/blob/main/CHANGELOG.md#0131-20230710">v0.13.1 Changelog</a>, and goes into effect with this release. You must update any code or Terraform configuration that uses the <code>id</code> parameter.
+    <br /><br />
+    Learn more:&nbsp;
+    <a href="/boundary/docs/commands/roles/add-grants">Add grants</a>
+    </td>
+  </tr>
+
+  <tr>
+    <td style={{verticalAlign: 'middle'}}>
+    0.15.0
+    </td>
+    <td style={{verticalAlign: 'middle'}}>
+    List result pagination
+    </td>
+    <td style={{verticalAlign: 'middle'}}>
+    All <code>list</code> endpoints now return the first 1,000 items instead of all items, if no parameters are provided. You can configure the number of items returned using the new controller configuration parameter <code>max_page_size</code>. The Admin UI, CLI, and API package automatically paginate results.
+    <br /><br />
+    Learn more:&nbsp;
+    <a href="/boundary/docs/api-clients/api/pagination">API list pagination</a>
+    </td>
+  </tr>
+
+  <tr>
+    <td style={{verticalAlign: 'middle'}}>
+    0.15.0
+    </td>
+    <td style={{verticalAlign: 'middle'}}>
+    Storage bucket policy updates
+    </td>
+    <td style={{verticalAlign: 'middle'}}>
+    If you have session recording configured, you must update your IAM policy in AWS before you upgrade to release 0.15.0 to ensure your session recordings continue to get uploaded to S3. The IAM policy now requires the <code>DeleteObjects</code> and <code>ListBucket</code> permissions.
```

Comment:
This policy change should happen before customers upgrade their worker to 0.15.0.